tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py27,py34,py35,py36,pypy,pypy3
+skipsdist = True
+
+[testenv]
+deps =
+	lxml!=4.2.0
+	pygost
+	pyyaml
+	py27,py34,py35,pypy: pyblake2
+	py27,py34,py35,pypy: pysha3
+setenv =
+	PYTHONPATH={toxinidir}/lib
+commands =
+	python setup.py test
+	python repoman/setup.py test
```

Comment:
How about if we use `python -b` to enable the BytesWarning when comparing str and bytes, like .travis.yml did before?

Reply:
Done. Also added `-Wd`.